packages/liveblocks-core/src/connection.ts

```diff
@@ -492,7 +490,10 @@ function createConnectionStateMachine<T extends BaseAuthResult>(
         if (failedEvent.reason instanceof StopRetrying) {
           return {
             target: "@idle.failed",
-            effect: log(LogLevel.ERROR, failedEvent.reason.message),
+            effect: [
+              log(LogLevel.ERROR, failedEvent.reason.message),
+              fireErrorEvent(failedEvent.reason.message, -1),
```

Comment:
@GuillaumeSalles Do you have a suggestion for the `-1` error code to use here? These aren't websocket codes, but the `code` param is mandatory on the `"error"` event. I used -1 here as a placeholder for now, but making it optional would technically be a breaking change.

Reply:
Related to [this](https://github.com/liveblocks/liveblocks/pull/1025#discussion_r1273705977), I think that if we type our error and add a new enum (different from `WebsocketCloseCodes`)  containing the codes, `-1` would be less visible.